querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx

```diff
@@ -232,6 +228,11 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
         return Object.keys(this.samplingTables).length > 0;
     }
 
+    public get sampleRate() {
+        // -1 for tables don't support sampling, 0 for default sample rate (which means disable sampling)
+        return this.hasSamplingTables ? this.state.meta.sample_rate ?? 0 : -1;
```

Comment:
@jczhong84  this cause max call stack size exceed since sampleRate() calls sampleRate() which calls this.sampleRate() again, I can revert it in my sampling PR if that sounds good to you